tests/vmi_networking_test.go

```diff
@@ -119,6 +119,16 @@ var _ = Describe("Networking", func() {
 		Expect(err).ToNot(HaveOccurred())
 	}
 
+	checkLearningState := func(vmi *v1.VirtualMachineInstance, expectedValue string, prompt string) {
+		err := tests.CheckForTextExpecter(vmi, []expect.Batcher{
```

Comment:
I think that you want to use `tests.RunCommandOnVmiPod`? That command you used is run inside the started VMI. Was that intentional?

Reply:
Nope good catch I'll fix.